### PR TITLE
Standardise a numbering style and indent level for lists

### PR DESCRIPTION
**Pull request by jgraham**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/pull/138.html" title="Last updated on Sep 23, 2021, 11:04 AM UTC (e1cc0f7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/138/ec58ee7...e1cc0f7.html" title="Last updated on Sep 23, 2021, 11:04 AM UTC (e1cc0f7)">Diff</a>

### CODE REVIEW

**Comment by foolip** [comment]
This conflicts now.

**Review by foolip — APPROVED** [review]
Approving so that it can be merged after resolving conflicts. I'll just keep conflicting for every PR merged :)

File: index.bs [hunk 1]
```diff
@@ -459,23 +459,23 @@ Note: |browsing contexts| is a set because a [=shared worker=] can be associated
...
+   [=error code=] [=Invalid Argument=].
 
-  1. Append the [=event name=] for each [=event=] in the module with name |name| to
-     |events|.
+1. Append the [=event name=] for each [=event=] in the module with name |name| to
+   |events|.
 
-  1. Return [=success=] with data |events|.
+1. Return [=success=] with data |events|.
 </div>
```
Comment (foolip) [inline index.bs:479]: Bikeshed wants different indentation here.

Comment (jgraham) [reply index.bs:479]: Not quite; if you have a non-indented list it turns out that you need a blank line before any markup following the list. Anyway, fixed.

**Pull request merged**